core/tests/performance_framework/process_data.py

```diff
@@ -0,0 +1,77 @@
+"""Script to process HAR data and timing data"""
```

Comment:
Can you spell out what 'HAR' is here, rather than using an acronym. Expect readers of this docstring to have no context on what this file is used for.